data/mods/MindOverMatter/snippets/snippets_new.json

```diff
@@ -61,7 +230,65 @@
+      },
+      {
+        "name": "LV-429 Analysis",
+        "id": "lab_file_matrix_research_04",
+        "text": "Human-portable laser weapons suffer from three major problems--high power draw preventing them from easily being transportable or usable in skirmish combat, long dwell times meaning that the operator must remain exposed to return fire in order to do anything more than blind the target, and common atmospheric conditions like dust or haze severely reducing the effectiveness of the beam.  The LV-429 solves all three of these problems.  Before the primary laser is fired, a secondary netherum mathemathical beam clears the path of the laser, removing atmospheric contaminants without affecting the target.  The laser is fired in a series of short pulses, each much higher intensity than a standard laser, eliminating the need for extensive dwell time.  Finally, the power source contains a small ND-cell, capable of powering the weapon effectively indefinitely.\n\nOne problem remains to be solved.  While the weapon is capable of providing enough power for every shot in succession, matrix technology has not yet provided us with more efficient battery storage.  This means that the ND-cell must charge existing batteries, allowing the LV-429 to be fired only once every few seconds, obviously too low a rate of fire for any live engagement.  Further enhancements in power storage technology will be necessary before proper deployment."
+      },
+      {
+        "name": "XE037 or whatever",
+        "id": "lab_file_matrix_research_05",
+        "text": "Yeah, we did the research.  We're all stuck here so what else can we do, right?  Just research our heart out while the world falls apart.  Well, I've attached all the figures so you can look at them, but the theory is right.  XE037 enhances exposed mathematicians (aka all of them) by a factor of anywhere from 10x to 100x or more, basically as much as using a matrix crystal.  Meshram used to be able to lift around 18.5g, one of our strongest telekinetics.  Now he's up to 2kg and with a crystal he can lift a two bowling balls.  Or a dog, that was hilarious.\n\nSubject U76 could pick up surface thoughts if touching the target and now can easily read someone's mind from across the room and can occasionally pick up deeper memories.  We figured this out by accident, by the way, and I'm pretty sure when we get out of here Dr. Connolly is going to find divorce papers waiting for her.  You'll probably want to keep U76 away from anything sensitive, except that we only just discovered this and now it's too late.  Oops.\n\nOh yeah, K34 can just walk through walls now.  So much for security.  Everything's fucked and we're doomed.  That's my report, hope you like it."
```

Comment:
I assume the phrase "can lift <ins>a</ins> two bowling balls" should be "can lift two bowling balls".